src/HealthCheckerAggregate.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Tseguier\HealthCheckBundle;
+
+
+use Tseguier\HealthCheckBundle\Dto\HealthData;
+
+class HealthCheckerAggregate implements HealthCheckInterface
+{
+    /**
+     * @var array|iterable
+     */
+    private $healthCheckers= [];
```

Comment:
```suggestion
    private $healthCheckers;
```
We are forcing developers to provide the iterable to construct this object anyway, no need for this (also, you can remove the. docblock)